posthog/udf_versioner.py

```diff
@@ -0,0 +1,70 @@
+import argparse
```

Comment:
Should it be a Django management script?

Reply:
I don't think there's any good reason to add that dependency, since these files have to run without django and it's really a completely separate subsystem.